src/Serilog.Sinks.MongoDB/project.json

```diff
@@ -0,0 +1,21 @@
+﻿{
+  "version": "2.0.0-beta-*",
+  "description": "The MongoDB sink for Serilog",
+  "authors": [ "Serilog Contributors" ],
```

Comment:
May be nice to add Kiran Makkapati explictly to this list since he was the original contributor of the code.